src/cmd/create.go

```diff
@@ -54,6 +56,8 @@ var (
 		{"/etc/profile.d/toolbox.sh", "/etc/profile.d/toolbox.sh"},
 		{"/etc/profile.d/toolbox.sh", "/usr/share/profile.d/toolbox.sh"},
 	}
+
+	enterCommand string
```

Comment:
Do we really need to make this a global variable? Usually, the fewer globals, the better.